RELEASENOTES.md

```diff
@@ -77,6 +77,39 @@ usage. (Intel IPU E2100 target only).
   * Supports maximum of 2 vports and each port supports maximum of 4 rx and tx
    queues
 
+### Kubernetes Infrastructure Offload Recipe
+
+#### Features and enhancements
+
+* Service Load Balancing: Support for K8s Service of type ClusterIP.
+   Kubeproxy implementation is now offloaded to hardware.
+   Services can be created and dynamically distributed to endpoints.
+   For TCP, only the SYN packet goes through the load-balancing logic.
+   Entry is added to hardware CT table for treatment of subsequent packets.
+   Support for dynamic scale-up of endpoints.
```

Comment:
Caching applies to both TCP and UDP. I would suggest change lines 86-88 to the following.
"TCP and UDP services can be created and are dynamically distributed to endpoints.
The first packet of each flow goes through load balancing logic, and the result is
cached in hardware auto-add table for treatment of subsequent packets, resulting in 
increased efficiency."